python/test/unit/tools/test_aot.py

```diff
@@ -435,7 +436,7 @@ def test_ttgir_to_ptx():
         kernel_path = os.path.join(tmp_dir, "empty_kernel.ttgir")
         with open(kernel_path, "w") as fp:
             fp.write(src)
-        k = triton.compile(kernel_path, target=("cuda", 80))
+        k = triton.compile(kernel_path, target=GPUTarget("cuda", 80, 32))
```

Comment:
Is this `80`, of type int, converted to str implicitly during the construction of GPUTarget?

Reply:
Nah, python is a dynamic language. Note that the type annotations to `GPUTarget` is more of type hints that are useful for readability and static analysis tools and such. It's not enforced by Python itself. 

I updated the type annotation to `Union[int, str]` to be clear though.